utils/sign-rulesets/async-request.sh

```diff
@@ -15,7 +15,8 @@ trap 'rm $SIGNED_SHA256SUM_BASE64' EXIT
 
 mkdir -p $2
 TIMESTAMP=`date +%s`
-echo "{ \"timestamp\": $TIMESTAMP, \"rulesets\":" "`cat $RULESETS_FILE`" "}" | tr -d '\n' | gzip -nc > $2/default.rulesets.$TIMESTAMP.gz
+REFERENCE=`git rev-parse HEAD`
+echo "{ \"timestamp\": $TIMESTAMP, \"reference\": \"$REFERENCE\", \"rulesets\":" "`cat $RULESETS_FILE`" "}" | tr -d '\n' | gzip -nc > $2/default.rulesets.$TIMESTAMP.gz
```

Comment:
Are the escaped quotes around `$REFERENCE` needed?

Reply:
The time stamp do not need an escape quote because it is an integer which is natively supported by the JSON format. The reference is an hex string, so I think it need an escaped quote.

Reply:
Looks good, please add this to `standalone.sh` as well and it'll be ready to merge.

Reply:
done. thanks!